### PR TITLE
OSDOCS-8701: Update resource considerations table

### PR DESCRIPTION
**Pull request by skrthomas**
For peer review/merge review:
Merge to only the no-1.5 branch - no cherrypicks are required.
This PR is part of an experiment for simplifying merges for asynchronous content, and I will open one PR against main to incorporate all of the Network Observability 1.5 content just before its GA.

Version(s):

4.11+
Issue:

https://issues.redhat.com/browse/OSDOCS-8701
Link to docs preview:

https://69821--docspreview.netlify.app/openshift-enterprise/latest/network_observability/configuring-operator#network-observability-resources-table_network_observability
QE review:
- [X] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by skrthomas** [comment]
@nathan-weinberg I know last week we discussed waiting on review for this update. I'm just going ahead and tagging you as a reviewer here now and you can review once you're sure that no other numbers will change for this table since from my side this is all that needs done AFAIK, for now.

File: modules/network-observability-resources-table.adoc [hunk 1]
```diff
@@ -17,7 +17,7 @@ The examples outlined in the table demonstrate scenarios that are tailored to sp
 |                                     | Extra small (10 nodes) | Small (25 nodes)  | Medium (65 nodes) ^[2]^ | Large (120 nodes) ^[2]^
 | *Worker Node vCPU and memory*       | 4 vCPUs\| 16GiB mem ^[1]^ | 16 vCPUs\| 64GiB mem ^[1]^ | 16 vCPUs\| 64GiB mem  ^[1]^  |16 vCPUs\| 64GiB Mem ^[1]^
 | *LokiStack size*                    | `1x.extra-small`         | `1x.small`          | `1x.small`           | `1x.medium`
-| *Network Observability controller memory limit* | 400Mi (default)        | 400Mi (default)   | 400Mi (default)    | 800Mi
+| *Network Observability controller memory limit* | 400Mi (default)        | 400Mi (default)   | 400Mi (default)    | 400Mi
```
Comment (nathan-weinberg) [inline modules/network-observability-resources-table.adoc:20]:
```suggestion
| *Network Observability controller memory limit* | 400Mi (default)        | 400Mi (default)   | 400Mi (default)    | 400Mi (default)
```

File: modules/network-observability-resources-table.adoc [hunk 2]
```diff
@@ -17,7 +17,7 @@ The examples outlined in the table demonstrate scenarios that are tailored to sp
...
 | *eBPF sampling rate*                | 50 (default)           | 50 (default)      | 50 (default)       | 50 (default)
 | *eBPF memory limit*                 | 800Mi (default)        | 800Mi (default)   | 2000Mi             | 800Mi (default)
```
Comment (nathan-weinberg) [inline modules/network-observability-resources-table.adoc:22]:
```suggestion
| *eBPF memory limit*                 | 800Mi (default)        | 800Mi (default)   | 800Mi (default)             | 1600Mi
```
@skrthomas might have messed up the spacing here but these are the new numbers :smile:

Comment (skrthomas) [reply modules/network-observability-resources-table.adoc:22]: @nathan-weinberg Thanks. Addressed!

**Comment by skrthomas** [comment]
@jotak Can you PTAL at these resource recommendation updates?

**Comment by nathan-weinberg** [comment]
I will approve pending @jotak's approval :wink:

**Review by jotak — APPROVED** [review]
lgtm!

**Review by skopacz1 — APPROVED** [review]
LGTM!

**Pull request merged**